linux_os/guide/services/usbguard/group.yml

```diff
@@ -5,4 +5,4 @@ title: 'USBGuard daemon'
 description: |-
     The USBGuard daemon enforces the USB device authorization policy for all USB devices.
 
-platform: not_s390x_arch
+platform: not_s390x_arch and machine
```

Comment:
As USBGuard rules should not be applicable to containers, I have included the additional conditions here, rather than the individual `rule.yml` files.